.travis.yml

```diff
@@ -8,6 +8,8 @@ os:
 addons:
   hosts:
     - mysql2gem.example.com
+env:
+  - MYSQL2_CI=1
```

Comment:
Looks like the alternate-DB envs in the matrix below are removing this. How about:
```
env:
  global:
    - MYSQL2_CI=1
```